examples/templates/example.html

```diff
@@ -165,7 +165,7 @@ <h5 class="source-heading">package.json</h5>
       </div>
     </div>
 
-    <script src="https://unpkg.com/elm-pep@1.0.6/dist/elm-pep.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/elm-pep@1.0.6/dist/elm-pep.js"></script>
```

Comment:
The others don’t use the cdn subdomain.  Curious if one is preferred over the other.